crates/parser2/src/ast/stmt.rs

```diff
@@ -63,14 +63,13 @@ ast_node! {
     SK::AssignStmt,
 }
 impl AssignStmt {
-    /// Returns the pattern of the lhs of the assignment.
-    pub fn pat(&self) -> Option<super::Pat> {
-        support::child(self.syntax())
+    /// Returns the expression of the lhs and rhs of the assignment.
+    pub fn lhs_expr(&self) -> Option<super::Expr> {
+        support::children(self.syntax()).next()
```

Comment:
Can I convert this to `support::child(...)`

Reply:
The result is the same. Also, it seems the performance wouldn't be different. Is there any motivation to change to `support::child`?

Reply:
just to keep the code concise. Also, with `next`, I feel it implies that it is pointing at the second item rather the first item, since it is not clear where the `iter` is starting (index -1 or 0).

Reply:
> I feel it implies that it is pointing at the second item rather the first item, since it is not clear where the iter is starting (index -1 or 0).
This is a well-known Rust idiom. Also, It'd be pretty strange if an iterator starts with the second element in any language, right?

When I wrote `support::children(...).next()`, I might feel it'd be slightly readable because `lhs` and `rhs` are symmetric(but I don't remember why, actually). 

Anyways, this is clearly a bikeshedding discussion. I'm perfectly fine with either.